src/app/find-help/reply-offer/reply-offer.component.html

```diff
@@ -51,7 +51,7 @@ <h3>{{ "REPLY_OFFER" | translate }}</h3>
           >
           ></textarea
           >
-          <mat-hint align="end">{{ data.text?.length || 0 }}/1000</mat-hint>
+          <mat-hint align="end">{{ data.text?.length || 0 }}/2000</mat-hint>
```

Comment:
a taka wskazówka:

 <textarea #textArea
 <mat-hint align="end">{{ data.text?.length || 0 }}/{{textArea.maxLength}}</mat-hint>
to powinno zadziałać, tylko dodaj prefix do # :)

Reply:
@bbartosiak ale to działa ;)

Reply:
dlatego wskazówka :)